client/src/pages/admin/NGOs.js

```diff
@@ -0,0 +1,161 @@
+import { useEffect, useState } from "react";
+import fileData from "./data/ngos.json";
+import Table from "react-bootstrap/Table";
+import Modal from "react-bootstrap/Modal";
+import Button from "react-bootstrap/Button";
+import Stack from 'react-bootstrap/Stack';
+import CreateNGO from "./components/CreateNGO";
+
+export default function NGOs() {
+  const [ngos, setNGOs] = useState([]);
+  const [singleNGO, setSingleNGO] = useState([]);
+  const [showFormModal, setShowFormModal] = useState(false);
+  const [showDeleteModal, setShowDeleteModal] = useState([false, 0]);
+  const [formAction, setFormAction] = useState("");
+
+  useEffect(() => {
+    const getNGOs = () => {
+      setNGOs(fileData);
+    }
```

Comment:
What do you think about sorting the data by organization names in alphabetical order?

  useEffect(() => {
    const getNGOs = () => {
      setNGOs(fileData.sort((a, b) => a.organization.localeCompare(b.organization)));
    };